blog/1.0-release/index.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: Announcing OpenLineage 1.0
+date: 2023-08-04
+authors: 
+    name: Shirley Lu
+    title: Guest Blogger
+    url: https://www.linkedin.com/in/juye-shirley-lu/
+image: ./image.svg
+banner: ./banner.svg
+description: We are pleased to announce the first major release of OpenLineage. This release completes the first version of a new static lineage feature.
+---
+
+Announcing OpenLineage [v1.0](https://github.com/OpenLineage/OpenLineage/releases/tag/1.0.0)! We’re officially in 1.x territory!
+
+![](./banner.svg#banner)
+
+<!--truncate-->
+
+It has become more and more apparent to us that managing data has become an O(n^2) problem. Every company is a data company, and every department within an organization is then again a mini data ecosystem in and of itself.  When departments interact, there’s a duplication of effort across pipeline tooling, and deployment of new tools can break existing lineage workflows. 
+
+This is why we’re seeing increasing demand for OpenLineage support, and we’re super excited to see more and more data engineers, developers and database managers joining our community. 
+
+Momentum seems to be growing behind OpenLineage as the dominant standard for runtime lineage collection. At the same time, a consensus appears to be forming that an open spec will get us closest to 100% coverage, one that is highly granular and also general-purpose enough to be consistent across various data workloads.
+
+### An Evolving Spec
+
+Now that we’re at v1.0, static lineage has made its way to OpenLineage! Traditionally, OpenLineage has focused on “runtime” lineage - i.e., metadata generated when jobs are run. Capturing information as transformations of datasets occur enables precise descriptions of those transformations. 
```

Comment:
Maybe `Up till now` rather than `Traditionally` - when speaking about traditions, I think of something more long lasting than 2-3 years.